feat: add music player app project (#52405)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/656071d679089ebd9d5035a0.md

---
id: 656071d679089ebd9d5035a0
title: Step 9
challengeType: 0
dashedName: step-9
---

# --description--

> Now, let's update the `userData` object to include a `songs` property. For this we will be using the `spread` operator.

Reviewer: ```suggestion
Now, you update the `userData` object to include a `songs` property. For this, you will be using the `spread` operator.
```
